epub33/core/index.html

```diff
@@ -691,320 +742,636 @@ <h4>Conformance Checking</h4>
-											<li><code>text/javascript</code></li>
-										</ol>
-									</td>
-									<td> [[RFC4329]] </td>
-									<td>Scripts.</td>
-								</tr>
-								<tr>
-									<td id="cmt-ncx">
-										<code>application/x-dtbncx+xml</code>
-									</td>
-									<td> [[OPF-201]] </td>
-									<td>The <a href="#legacy">legacy</a> NCX.</td>
-								</tr>
-								<tr>
-									<td id="cmt-smil">
-										<code>application/smil+xml</code>
-									</td>
-									<td>
-										<a href="#sec-media-overlays">Media Overlays</a>
-									</td>
-									<td>EPUB Media Overlay documents</td>
-								</tr>
-							</tbody>
-						</table>
+							<li>Otherwise, it MAY reference Foreign Resources in its <code>src</code> and
+									<code>srcset</code> attributes provided EPUB Creators define a <a
+									href="#sec-manifest-fallbacks">manifest fallback</a>.</li>
+						</ul>
 					</section>
+				</section>
 
-					<section id="sec-foreign-restrictions">
-						<h5>Foreign Resources</h5>
+				<section id="sec-exempt-resources">
+					<h4>Exempt Resources</h4>
+
+					<p>An <a>Exempt Resource</a> shares properties with both <a>Foreign Resources</a> and <a>Core Media
+							Type Resources</a>. It is most similar to a <a>Foreign Resource</a> in that it is not
+						guaranteed <a>Reading System</a> support, but, like a Core Media Type Resource, does not require
+						a fallback.</p>
+
+					<p>There are only a small set of special cases for Exempt Resources. Video, for example, are exempt
+						from fallbacks because there is no consensus on a Core Media Type video format at this time
+						(i.e., there is no format to fallback to). Similarly, audio and video tracks are exempt to allow
+						EPUB Creators to meet accessibility requirements using whatever format Reading Systems support
+						best.</p>
+
+					<p>The various types of Exempt Resources are defined in the following list, including any
+						restrictions on where EPUB Creators can use them:</p>
+
+					<dl>
+						<dt id="exempt-fonts">Fonts</dt>
+						<dd id="confreq-resources-cd-fonts">
+							<p>All font resources not already covered as <a href="#cmt-grp-font">font Core Media
+									Types</a> are Exempt Resources.</p>
+							<p>This exemption allows EPUB Creators to use any font format without a fallback, regardless
+								of Reading System support expectations, as CSS rules will ensure a fallback font in case
+								of no support.</p>
+							<p>Refer to the <a data-cite="epub-rs-33#confreq-css-rs-fonts">Reading System support
+									requirements for fonts</a> [[EPUB-RS-33]] for more information.</p>
+						</dd>
 
-						<p id="confreq-foreign-no-fallback">EPUB Creators MAY include <a>Foreign Resources</a> without
-							fallbacks provided they:</p>
+						<dt id="exempt-links">Linked resources</dt>
+						<dd id="confreq-resources-cd-fallback-link">
+							<p>Any resource referenced from the [[HTML]] <a data-cite="html#the-link-element"
```

Comment:
I think that, just as in the case of Fonts, we should say "All resources not already covered Core Media Types...". Otherwise CSS, used via a link element, has a dual categorization...

Maybe a general statement for all exempt resources may be better to cover all cases.

Reply:
Yes, good catch!

I tried to come up with a general explanation, but it really only applies to fonts and html links. I found it much clearer just to clearly exclude CMTs in both cases. My best attempts at a general explanation sound like a lot of confusing bloviation, as the exemptions aren't consistent (fonts aren't tied to a specific mechanism while links are).

The latest commit adds an exclusion on CMTs and also includes a link from the intro down to the new example.